ui/Dockerfile

```diff
@@ -3,14 +3,13 @@ FROM python:3.7.4-stretch
 # RUN apt-get update && apt-get install -y curl git pkg-config cmake
 
 # copy code
-RUN mkdir ui/
-COPY . /opt/ui
+COPY . /ui
 
 # install as a package
 RUN pip install --upgrade pip && \
-    pip install /opt/ui/
+    pip install /ui/
 
-WORKDIR /opt
```

Comment:
Just curious here, there was a specific reason to move `/ui` out of `/opt`?

Reply:
No particular reason.
I tried to clean and simplify the Dockerfile.
If it's not desirable, I can easily revert this change!

Reply:
No no, that's all right :+1: I'll tag @masci just in case but I don't see an issue here.

Reply:
Not a big deal, `/opt` is just a convention (albeit a very widespread one 😅)